sre-agent-xpander.ai/sre_agent.py

```
@@ -0,0 +1,108 @@
+# look for any kubectl command and strip code fences
+KUBECTL = re.compile(r"kubectl\s+(.+)", re.IGNORECASE)
+FENCE   = re.compile(r"```[\s\S]*?```", re.MULTILINE)
+
+class LocalKubectlTool(MultiMCPTools):
+    name = "kubectl"
+
+    def __init__(self) -> None:
+        super().__init__([self.name], env={})
+        # capture context once
+        self.ctx = subprocess.run(
+            ["kubectl","config","current-context"],
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)